api/package.json

```diff
@@ -199,6 +199,7 @@
 		"@types/node-schedule": "2.1.0",
 		"@types/nodemailer": "6.4.7",
 		"@types/object-hash": "3.0.2",
+		"@types/papaparse": "^5.3.8",
```

Comment:
Let's pin the specific version, like the other dependencies

Reply:
Done